mtest/libmscore/midi/tst_midi.cpp

```diff
@@ -55,6 +55,7 @@ class TestMidi : public QObject, public MTest
       void midiBendsExport1() { midiExportTestRef("testBends1"); }
       void midiBendsExport2() { midiExportTestRef("testBends2"); }      // Play property test
       void midiPortExport()   { midiExportTestRef("testMidiPort"); }
+      void mediArpeggio()     { midiExportTestRef("arpeggio"); }
```

Comment:
typo, should be `midiArpeggio()`

Reply:
thanks

Reply:
And probably also `testArpeggio`, so the files should be named testArpeggio.mscx and testArpeggio-ref.mid